src/Traits/ReflectionFunctionLikeTrait.php

```diff
@@ -63,33 +70,59 @@ public function getClosureThis()
     {
         $this->initializeInternalReflection();
 
-        return forward_static_call('parent::getClosureThis');
+        return parent::getClosureThis();
     }
 
     public function getDocComment()
     {
+        if (!$this->functionLikeNode) {
```

Comment:
I see lots of similar checks for internal reflection initialization. Maybe we can create a proxy classes that would use `__call` method where optional initialization of internal reflection would be made.

That would greatly reduce copy/paste of such code, but I'm not sure if excessive usage of `__call` method would result in performance decrease.

Reply:
I'm thinking similarly. I believe `__call()' is pretty expensive, but I believe this can be simplified. I'll look into it further. :+1:

Reply:
I have some thoughts on this I want to explore. No promises, but we'll see.

Reply:
OK.